go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/dims/community-images
 
-go 1.15
+go 1.18
```

Comment:
Can we use go 1.19 instead?

Reply:
If desired I can of course use go1.19. I only took the minimal version that might solve the errors.